beeline/src/main/resources/beeline-log4j2.properties

```diff
@@ -33,12 +33,16 @@ appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{yy/MM/dd HH:mm:ss} [%t]: %p %c{2}: %m%n
 
 # list of all loggers
-loggers = HiveConnection
+loggers = HiveConnection, HiveStatement
 
 # HiveConnection logs useful info for dynamic service discovery
 logger.HiveConnection.name = org.apache.hive.jdbc.HiveConnection
 logger.HiveConnection.level = INFO
 
+# HiveStatement logs for tracking the running statement, such as the operation's drilldown link
+logger.HiveStatement.name = org.apache.hive.jdbc.HiveStatement
```

Comment:
Are we sure we want to add more logging to Beeline? @pvary do we have any general guidelines on what could/should be logged in beeline?

Reply:
Most of the logs in HiveStatement are in debug level,  showing the process of excuting a statement. Here enable it because we want to enable showing the operation display link at the beginning of executing the statement when using beeline.

Reply:
@dengzhhu653: I think showing the link should be decided based on this method:
```
if (sessionManager.getOperationManager().canShowDrilldownLink(operationHandle)) {
```
Are you saying that this is not working as expected? Even if the config is enabled and the line is appended to the info messages then it is not displayed on BeeLine?

Reply:
@pvary Yes, I tried on local test, the link is displayed on beeline only if the config is enabled as well as adding the HiveStatement(loggers) to the beeline-log4j2.properties.

Reply:
@dengzhhu653: There are some users who have hard expectations on the stdout/stderr for BeeLine processes. I think we should keep the original behavior here. If someone is interested they can change the config on their own deployment.

Thanks,
Peter

Reply:
Fine, I moved the other minor parts into the https://github.com/apache/hive/pull/1992,  could you please take a look at the PR-1992? Thank you...

Reply:
@dengzhhu653: Please do mix fixes / changes in PRs only if necessary. This helps backport work easier. If you could remove the log config changes from this one, then I could merge this PR, and we can continue to the next one.

Thanks, Peter

Reply:
ok, done.